packages/itwinui-react/src/core/utils/components/PolymorphicAction.tsx

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import {
+  PolymorphicComponentProps,
+  PolymorphicForwardRefComponent,
+} from '../index';
```

Comment:
we should never import anything from `index` to avoid risking circular dependencies.

```suggestion
} from '../props';
```

Reply:
importing from props.